packages/flutter/test/animation/live_binding_test.dart

```diff
@@ -77,7 +78,9 @@ void main() {
     // Currently skipped due to daily flake: https://github.com/flutter/flutter/issues/87588
   }, skip: true); // Typically skip: isBrowser https://github.com/flutter/flutter/issues/42767
 
-  testWidgets('Should show event indicator for pointer events with setSurfaceSize', (WidgetTester tester) async {
+  testWidgets('Should show event indicator for pointer events with setSurfaceSize',
+  experimentalLeakTesting: LeakTesting.settings.withIgnoredAll(),
```

Comment:
For other ignored things (lints, skipped tests, etc) we require comments stating a reason for the ignore to give context of whether this is supposed to be ignored indefinitely or whether this is only due to a bug that should eventually get fixed to remove the ignore. Can we establish the same pattern for ignored leaks?